src/custom/utils/operator/error.ts

```diff
@@ -17,13 +17,17 @@ export interface ApiError {
 }
 
 const API_ERROR_CODE_DESCRIPTIONS = {
-  [ApiErrorCodes.DuplicateOrder]: 'There was another identical order already submitted',
-  [ApiErrorCodes.InsufficientFee]: "The account doesn't have enough funds",
-  [ApiErrorCodes.InvalidSignature]: 'The order signature is invalid',
+  [ApiErrorCodes.DuplicateOrder]: 'There was another identical order already submitted. Please try again.',
+  [ApiErrorCodes.InsufficientFee]:
+    "The signed fee is insufficient. It's possible that is higher now due to a change in the gas price, ether price, or the sell token price. Please try again so you get an updated fee quote.",
```

Comment:
```suggestion
    "The signed fee is insufficient. It's possible that is higher now due to a change in the gas price, ether price, or the sell token price. Please try again to get an updated fee quote.",
```

Reply:
```suggestion
    "The signed fee is insufficient. It may be possible that the fee is higher now due to a change in the gas, ether, or sell token price. Please try again to get an updated fee quote.",
```